docset/winserver2019-ps/printmanagement/Set-PrintConfiguration.md

```diff
@@ -77,7 +77,7 @@ The final command sets the value of the printer configuration to the modified va
 
 ### Example 3: Set the default paper size of all printers
 ```
-PS C:\>$Printers = Get-Printer * Foreach ($Printer in $Printers){     Set-PrintConfiguration -PrinterName $Printer.name -PaperSize A4}
+PS C:\>$Printers = Get-Printer ; Foreach ($Printer in $Printers){     Set-PrintConfiguration -PrinterName $Printer.name -PaperSize A4}
```

Comment:
```suggestion
PS C:\> $Printers = Get-Printer | Foreach ($Printer in $Printers){Set-PrintConfiguration -PrinterName $Printer.name -PaperSize A4}
```

Reply:
actually, the ";" is ok here, it allows for multiple statements on a single line

Reply:
It's true